### ISSUE TITLE
[solid-js] Not found: type T

### ISSUE THREAD

**Issue by user753** [ScalablyTyped/Converter#425]
I am trying to use Solidjs
```
Compile / npmDependencies ++= Seq(
	  "solid-js" -> "1.3.0"
)
```
But I get this error
```
[error] 19 |  inline def InitialValue(initialValue: T): typings.solidJs.anon.InitialValue = {
[error]    |                                        ^
[error]    |                                        Not found: type T]) 
```

**Comment by oyvindberg** [comment]
Thanks for the report. Looks like it should be easy enough to fix :)

**Comment by oyvindberg** [comment]
Fixed in 9b1ecbb611dde1b5bfd30b44368ff57d4399a175 .

**Comment by user753** [comment]
Thanks. When will It be released?
